docs/community/governance.md

```diff
@@ -0,0 +1,180 @@
+In the body, the call to vote must state the proposal being voted on. It should reference any discussion leading up to this point.
+
+Votes may take the form of a single proposal, with the option to vote yes or no, or the form of multiple alternatives.
+
+A vote on a single proposal is considered successful if more vote in favor than against.
+
+If there are multiple alternatives, members may vote for one or more alternatives, or vote “no” to object to all alternatives.
+It is not possible to cast an “abstain” vote. A vote on multiple alternatives is considered decided in favor of one alternative if it has received the most votes in favor, and a vote from more than half of those voting. Should no alternative reach this quorum, another vote on a reduced number of options may be called separately.
+
+### Supermajority vote
+
+Supermajority votes must be called explicitly in a separate thread on the appropriate mailing list.
+The subject must be prefixed with `[VOTE]`. In the body, the call to vote must state the proposal being voted on. It should reference any discussion leading up to this point.
+
+Votes may take the form of a single proposal, with the option to vote yes or no, or the form of multiple alternatives.
+
+A vote on a single proposal is considered successful if at least two thirds of those eligible to vote vote in favor.
+
+If there are multiple alternatives, members may vote for one or more alternatives, or vote “no” to object to all alternatives.
+A vote on multiple alternatives is considered decided in favor of one alternative if it has received the most votes in favor, and a vote from at least two thirds of those eligible to vote. Should no alternative reach this quorum, another vote on a reduced number of options may be called separately.
+
+## FAQ
+
+This section is informational. In case of disagreement, the rules above overrule any FAQ.
+
+### For majority vote, what if there is even number of maintainers and an equal amount of votes in favor than against?
+
+It has to be majority so the vote will be declined.
+
+### So what's the TLDR difference between majority vs supermajority?
+
+It's about number of up votes to agree on the decision.
+
+* majority: Majority of voters has to agree.
+* supermajority: 2/3 voters has to agree.
+
+### How do I propose a decision?
+
+See [Contributor doc](/CONTRIBUTING.md#adding-new-features--components)
+
+### How do I become a team member?
+
+To become an official member of Maintainers Team, you should make ongoing contributions to one or more project(s) for at least three months.
+At that point, a team member (typically a maintainer of the project) may propose you for membership.
+The discussion about this will be held in private, and you will be informed privately when a decision has been made. A possible, but not required, graduation path is to become a maintainer first.
```

Comment:
should this be triage? Otherwise becoming a maintainer is part of the path of becoming a maintainer?
```suggestion
The discussion about this will be held in private, and you will be informed privately when a decision has been made. A possible, but not required, graduation path is to become a triage member first.
```